pkg/build/build.go

```diff
@@ -0,0 +1,45 @@
+// Package build provides details of the built binary
+// The details are set using ldflags.
+//
+// The ldflags can be set either manually:
+// `go build -ldflags "-X github.com/gomods/athens/pkg/build.commitSHA=$(git rev-list -1 HEAD) -X github.com/gomods/athens/pkg/build.version=$(git describe --tags) -X github.com/gomods/athens/pkg/build.buildDate=$(date -u +%Y-%m-%d-%H:%M:%S-%Z)"`
+//
+// or using the build script in ./scripts.
+package build
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// details represents known data for a given build
+type details struct {
+	Version string
+	Commit  string
```

Comment:
Just my opinionbut I think Version will be enough - if there is no tag than use commit hash otherwise tag.